index.html

```diff
@@ -1002,6 +1002,39 @@ <h2 class='informative'>Implementation Patterns</h2>
       </section>
     </section>
 
+    <section id="privacy">
+      <h2>Privacy Considerations</h2>
+      <p>
+        Fedora resources may contain personally identifiable information such as the identify of
+        the agents who create and modify resources. As such, this information could be analyzed
+        to generate profiles of the behavior of individuals or groups.
+      </p>
+      <p>
+        As such, Fedora implementations that produce or consume personally identifiable
+        information MUST clearly document the type and extent of personal information that
+        is collected and published along with the reasons for doing so.
```

Comment:
@ajs6f much of what is said in the privacy section is there because of what exists in the privacy section of the [Activity Streams specification](https://www.w3.org/TR/activitystreams-core/#privacy) (which the Fedora spec makes use of):

> Implementations that produce or consume Activity Streams 2.0 documents must take steps to openly and publicly document and communicate to all potential users: (a) the kinds of potentially sensitive personal information published, consumed or collected by the implementation, (b) the reasons for publishing, consuming and collecting that information, (c) the manner in which that information is being used, (d) the identity of any other party with whom that information is being shared, and (e) the reason the information is being shared with other parties.
> 
> Implementations that publish Activity Streams 2.0 documents should assume a default position of limiting both the kind and amount of sensitive personal information included in the document unless users have "opted in" to sharing additional detail.

Given that a complying Fedora implementation would be an activity stream producer, it would necessarily follow that that implementation `MUST` include some documentation about any privacy considerations.

Alternately, Fedora could ignore the Activity Streams `MUST` statement, perhaps with it's own softer statement (e.g. `MAY`), but I would advocate for conforming with Activity Streams.